app/renderer/components/common/messageBox.js

```diff
@@ -154,7 +158,10 @@ const styles = StyleSheet.create({
     marginTop: globalStyles.spacing.dialogInsideMargin,
     minWidth: '425px',
     marginBottom: globalStyles.spacing.dialogInsideMargin,
-    userSelect: 'text'
+    userSelect: 'text',
+    maxHeight: '300px',
```

Comment:
Oh, good idea @luixxiul with `vh`. Since dialogs are pretty much always sized and positioned relatively to the viewport, it makes sense to use `vh` and `vw`. What do you think, @cezaraugusto?